packages/e2e/cypress/integration/common/shared.ts

```diff
@@ -270,6 +276,10 @@ When('I click the {string}', (id: string) => {
   cy.findByTestId(id).click();
 });
 
+When('I click the element with id attribute {string}', (id: string) => {
```

Comment:
An alternative to this is `I click the {string}` that uses `cy.findByTestId`. This requires the element to have `data-testid` prop.

Adding to prop to dynamic components like table might be complex hence introduced this as an alternative.

Reply:
Nit: should click always have `force` set to true when we want to select by id? And would it make sense to allow any selector rather than an id? Or would that be too generic?

Reply:
>Nit: should click always have force set to true when we want to select by id?

We're using this to select the table header checkbox input which is hidden so force is required in this scenario. 

>would it make sense to allow any selector rather than an id? Or would that be too generic?

Yeah i think it might too generic, we could add one later if required

Reply:
> We're using this to select the table header checkbox input which is hidden so force is required in this scenario.

I was thinking if this could be an optional choice for situations where we want to select by an id and _not_ force the click. But now that I think of it, I can't think of a scenario where forcing the click in this circumstance would be unwanted.

As you say, it can be updated later if additional test cases require more flexible behavior.